Breeding-Insight/bi-api
# [BI-2046] Overwrite observations happening without reference to ObsUnitID

# Description
**Story:** [BI-2046](https://breedinginsight.atlassian.net/jira/software/c/projects/BI/boards/1?selectedIssue=BI-2046)

Validation errors are now returned for display in the experiment import preview for any rows referring to an existing environment and with the ObsUnitID blank.

# Dependencies
bi-web release/0.9

# Testing
Verify all tests pass in ExperimentFileImportTest
1) import an experiment with observations
2) download the exp data with ObsUnitId populated
3) select multiple observations for updating and change phenotype values
4) import and verify updates by viewing experiment details

# Checklist:



[BI-2046]: https://breedinginsight.atlassian.net/browse/BI-2046?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
